native/src/components/SettingItem.tsx

```diff
@@ -13,25 +12,36 @@ const PadView = styled.View`
   align-items: center;
   background-color: ${props => props.theme.colors.backgroundColor};
 `
-const RightContentContainer = styled.View`
-  flex: 0.4;
-  justify-content: center;
-  align-items: flex-end;
-  padding: 0 5px;
-`
+
 const ContentContainer = styled.View`
   flex: 1;
   justify-content: center;
   align-items: flex-start;
 `
+
 const Title = styled.Text<{ bigTitle: boolean }>`
   color: ${props => props.theme.colors.textColor};
   ${props => (props.bigTitle ? 'font-size: 18px;' : '')}
 `
+
 const Description = styled.Text`
   color: ${props => props.theme.colors.textSecondaryColor};
 `
 
+const RightContentContainer = styled.View`
+  flex: 0.4;
+  justify-content: center;
+  align-items: flex-end;
+  padding: 0 5px;
+`
+
+const Badge = styled.View<{ enabled: boolean }>`
+  width: 8px;
+  height: 8px;
+  border-radius: 4px;
+  background-color: ${props => (props.enabled ? 'limegreen' : 'red')};
```

Comment:
Could we add the colors to the constants and use hex codes?

Reply:
Hmmm, these are pretty random colors I picked to roughly match the previous ones. The design system doesn't have feedback colors yet, I'd rather wait for those.

Reply:
For me actually an even bigger reason to add it to the constants. Then we can easily change it later once we have the correct colors and don't just forget it.

Reply:
I think I'll leave it because they can easily found by search for red or green in the code, and I assume that will happen then.